<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="prefCategories">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDatabase">
      <attribute name="title">
       <string>Databse</string>
      </attribute>
      <widget class="QGroupBox" name="gbPassType">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>302</width>
         <height>151</height>
        </rect>
       </property>
       <property name="title">
        <string>Passport Type for Image Searching</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rbLumHist">
          <property name="text">
           <string>Luminance Histogram</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbSubLum">
          <property name="text">
           <string>Subsampled Luminance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbLumGradHist">
          <property name="text">
           <string>Luminance and Gradient Histigram</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbSubLumGrad">
          <property name="text">
           <string>Subsampled Luminance and Gradient</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabColorization">
      <property name="toolTip">
       <string/>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <attribute name="title">
       <string>Colorization</string>
      </attribute>
      <widget class="QGroupBox" name="gbColorMethod">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>271</width>
         <height>181</height>
        </rect>
       </property>
       <property name="title">
        <string>Colorization Methods</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbWSimple">
          <property name="text">
           <string>Walsh Simple</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbWSLookUp">
          <property name="text">
           <string>Walsh Simple (look up table)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbWSEntropy">
          <property name="text">
           <string>Walsh Simple (entropy)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbWNeighbor">
          <property name="text">
           <string>Walsh Neighbor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbWNNoRand">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Walsh Neighbor (fixed net)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbWNOneRand">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Walsh Neighbor (same pixels set)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="gbImgLumEqual">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>301</width>
         <height>181</height>
        </rect>
       </property>
       <property name="title">
        <string>Images Luminance Equalization</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="rbNoScale">
          <property name="text">
           <string>No Scale</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbScaleMaxLum">
          <property name="text">
           <string>Scale by Max Luminance</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbScaleAverLum">
          <property name="text">
           <string>Scale by Average Luminance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbNormLumBorder">
          <property name="text">
           <string>Normalize Luminance (by borders)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbNormLumCentral">
          <property name="text">
           <string>Normalize Luminance (by center)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="exitBBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitBBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitBBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
